anago

```diff
@@ -203,25 +203,29 @@ check_prerequisites () {
        (logrun gcloud config set account $user && \
         logrun gcloud docker -- version >/dev/null 2>&1); then
       logecho -r "$OK"
+      break
```

Comment:
Yes, the loop validates both users (if set), so break'ing here is not what we want to do.